go.mod

```diff
@@ -1,6 +1,7 @@
 module github.com/prometheus/blackbox_exporter
 
 require (
+	github.com/andybalholm/brotli v1.0.0
```

Comment:
this is needed to support decompressing `br`

Reply:
This is no longer needed.

Reply:
I need this for the tests.

I thought about just reporting `Content-Encoding: br` and not providing a body that's actually compressed using that, but the intention behind leaving those tests there is to catch behavior changes in underlying libraries (like automatically decompressing content) so I need to provide something that is actually compressed like that.

Reply:
Let's not be pulling in dependencies just for tests of things we don't support.

Reply:
OK, done